src/Discord/Parts/WebSockets/MessageReaction.php

```diff
@@ -261,7 +261,7 @@ public function delete(?int $type = null): ExtendedPromiseInterface
                 break;
             case Message::REACT_DELETE_ID:
             default:
-                if (! $userid = $this->user_id ?? $this->user->id) {
+                if (!$userid = $this->user_id ?? $this->user->id) {
```

Comment:
obsolete change

Reply:
That's not the issue...

This is to deal with the fact that the response for MESSAGE_REACTION_REMOVE includes emoji, but the response for MESSAGE_REACTION_REMOVE_ALL does not.


`[2023-12-29 02:51:02] local.ERROR: Undefined array key "emoji" {"exception":"[object] (ErrorException(code: 0): Undefined array key \"emoji\" at /Volumes/ExtremeSSD/workspace/Herd/430test/packages/team-reflex/discord-php/src/Discord/Parts/WebSockets/MessageReaction.php:223)
[stacktrace]
#0 /Volumes/ExtremeSSD/workspace/Herd/430test/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(255): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleError(2, 'Undefined array...', '/Volumes/Extrem...', 223)
#1 /Volumes/ExtremeSSD/workspace/Herd/430test/packages/team-reflex/discord-php/src/Discord/Parts/WebSockets/MessageReaction.php(223): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->Illuminate\\Foundation\\Bootstrap\\{closure}(2, 'Undefined array...', '/Volumes/Extrem...', 223)
#2 /Volumes/ExtremeSSD/workspace/Herd/430test/packages/team-reflex/discord-php/src/Discord/Parts/Part.php(239): Discord\\Parts\\WebSockets\\MessageReaction->getEmojiAttribute()
#3 /Volumes/ExtremeSSD/workspace/Herd/430test/packages/team-reflex/discord-php/src/Discord/Parts/Part.php(388): Discord\\Parts\\Part->getAttribute('emoji')
#4 /Volumes/ExtremeSSD/workspace/Herd/430test/packages/team-reflex/discord-php/src/Discord/Parts/Part.php(533): Discord\\Parts\\Part->getPublicAttributes()
#5 /Volumes/ExtremeSSD/workspace/Herd/430test/vendor/monolog/monolog/src/Monolog/Logger.php(605): Discord\\Parts\\Part->__toString()
#6 /Volumes/ExtremeSSD/workspace/Herd/430test/modules/Discord/app/Console/WebSocket.php(287): Monolog\\Logger->info(Object(Discord\\Parts\\WebSockets\\MessageReaction))
#7 /Volumes/ExtremeSSD/workspace/Herd/430test/vendor/evenement/evenement/src/EventEmitterTrait.php(143): Modules\\Discord\\app\\Console\\WebSocket->Modules\\Discord\\app\\Console\\{closure}(Object(Discord\\Parts\\WebSockets\\MessageReaction), Object(Discord\\Discord), NULL)
#8 /Volumes/ExtremeSSD/workspace/Herd/430test/packages/team-reflex/discord-php/src/Discord/Discord.php(779): Discord\\Discord->emit('MESSAGE_REACTIO...', Array)
`